G2P.pm

```diff
@@ -397,6 +424,19 @@ sub run {
   return $results;
 }
 
+=head2 set_variant_include_list_flag
+
+  Arg [1]    : TranscriptVariationAllele $tva
+  Description: Check if variant is part of the variant include list.
+               If the variant is on the variant include list then report
+               it in the result set regardless if the variant passed the
+               filtering by variant consequence and allele frequency.
+  Returntype : None
+  Exceptions : None
+  Caller     : General
+  Status     : Stable
+
+=cut
 sub set_variant_include_list_flag {
```

Comment:
Inside this method, it checks if `$self->{user_params}->{variant_include_list}` is defined. This check could be done before `set_variant_include_list_flag` is called in L412.